sdks/python/apache_beam/io/avroio_test.py

```diff
@@ -25,10 +25,15 @@
 from builtins import range
 
 import avro.datafile
-import avro.schema
 from avro.datafile import DataFileWriter
 from avro.io import DatumWriter
 import hamcrest as hc
+# pylint: disable=wrong-import-order, wrong-import-position, ungrouped-imports
+try:
+  from avro.schema import Parse
```

Comment:
Could you add a comment here about, in what versions of avro which version of parse is supported. (We can use this information to remove this block later on.)

Reply:
+1 to @aaltay's comment. Also we will need a similar change in other places in Beam where we use avro.schema.parse. See `apache_beam/io/avroio.py`, `apache_beam/examples/avro_bitcoin.py`. This can be done in another PR if you prefer.

Reply:
also `apache_beam/examples/fastavro_it_test.py`.

Reply:
Thank you both for the feedback.

@aaltay I have added a comment stating that python2 uses the 'avro' library, whereas python3 uses 'avro-python3'.
@tvalentyn I have made the changes in the related files as well.

Note that although I had disabled pylint wrong-import-order and wrong-import-position, it still did not pass the precommit check so I had to reposition the imports to continue, PTAL, thanks!